src/ui/js/Overview.js

```diff
@@ -530,11 +530,12 @@ Overview.pageAddIntroText = function() {
       'This is the <span style="color: red;">' +
       Config.siteType.toUpperCase() +
       '</span> version of ' +
-      'the Buttonweavers implementation of ');
-    infopar.append($('<a>', {
-      'href': 'https://cheapass.com/free-games/button-men/',
-      'text': 'Button Men',
-    }));
+      'the Buttonweavers implementation of Button Men');
+//      'the Buttonweavers implementation of ');
+//    infopar.append($('<a>', {
+//      'href': 'https://cheapass.com/free-games/button-men/',
+//      'text': 'Button Men',
+//    }));
```

Comment:
I'd rather not deal with this by hand --- it's literally going to be by hand.

Let's just change https to http, and then it'll redirect to *something*.